ppb/sprites.py

```diff
@@ -144,7 +145,8 @@ class BaseSprite(EventMixin):
     image = None
     resource_path = None
 
-    def __init__(self, size: int=1, pos: Iterable=(0, 0), blackboard: Dict=None, facing: Vector=Vector(0, -1)):
+    def __init__(self, size: Union[int, float]=1, pos: Iterable=(0, 0),
+                 blackboard: Dict=None, facing: Vector=Vector(0, -1)):
```

Comment:
Yeah, that was imagined as a hidden element, and it's. . . yeah not useful. It might just be size.

I'm locking it to height mostly to pick a thing? Game Objects are still square and/or circles depending on what we do with it, and so images that aren't square, you just. . . pick one.

Reply:
`width` was the default in my mind, but I feel like just pick one and call it `sprite_height` or just `height` or something.

Assuming we don't feel like writing the code to allow it to fix on either width or height. (Like, of course we could have that flexibility, it just sounds tedious an repetitive to write.)

Reply:
Size is the size of the bounding box, which is square. The sprite images are resized based on height because most of the resources I've gathered are either square or sprites that are wider than tall and with details in the width that are okay to be "safe" areas as far as collision checking goes. (The lasers being the only ones that don't fit the pattern.)

Reply:
I can rewrite it to scale to shortest side and we can note that it's encouraged to use things "close" to square.